### DISCUSSION TITLE
Feature request: a feature to jump to errors like in Emacs compilation mode

### DISCUSSION THREAD

**Discussion by rakivo** [Zeioth/compiler.nvim#31] [Ideas]
Thank you guys for such a beautiful plugin!

 Just wanted to tell you that if it's possible to implement [Compilation mode](https://www.gnu.org/software/emacs/manual/html_node/emacs/Compilation-Mode.html), that would be so awesome. Love y'all!

**Comment by Zeioth** [comment]
Ideally that should be a different plugin.
